src/lib/Commands/TestCommand/index.js

```diff
@@ -268,22 +281,30 @@ class TestCommand extends AbstractCommand {
     });
   }
 
-  /**
-   * Prepare source code
-   * @param testFile
-   * @return {{agent: string, device: string}}
-   * @private
-   */
+  // Prepare source code
+  // @param testFile
+  // @return {{agent: string, device: string}}
+  // @private
+
   _getSessionCode(testFile) {
     let agentCode, deviceCode;
 
-    /* [info] */
+    // [info]
     this._info(c.blue('Using ') + testFile.type + c.blue(' test file ') + testFile.name);
 
     // read/process test code
-    let testCode = fs.readFileSync(testFile.path, 'utf-8').trim();
-    this._codeProcessor.variables.__FILE__ = path.basename(testFile.path);
-    testCode = this._codeProcessor.process(testCode);
+    let testCode = fs.readFileSync(testFile.path, 'utf-8').trim().replace(LINE_AND_FILE_REGEXP, "@{__");
```

Comment:
Cool the code looks much better and cleaner to me know! Thanks.

The only think I would do is to explicitly do the replacement for __FILE__/__PATH__, rather than for anything that starts with __.